src/main/java/BobBot.java

```diff
@@ -0,0 +1,104 @@
+import java.util.Scanner;
+
+public class BobBot {
+
+    private static Task[] allTasks = new Task[100];
+    private static int numTasks = 0;
+
+    private static void mark(String line) {
+        int taskNumToMark = Integer.parseInt(line.substring(4).trim()) - 1;
+        allTasks[taskNumToMark].markAsDone();
+        drawLine(true);
+        System.out.println("\tGot it! Marking this task as done:");
+        System.out.printf("\t[%s] %s\n", allTasks[taskNumToMark].getStatusIcon(),
+                allTasks[taskNumToMark].getDescription());
+        drawLine(true);
+    }
+
+    private static void unmark(String line) {
+        int taskNumToUnmark = Integer.parseInt(line.substring(6).trim()) - 1;
+        allTasks[taskNumToUnmark].markAsUndone();
+        drawLine(true);
+        System.out.println("\tAlright! Unmarking this task:");
+        System.out.printf("\t[%s] %s\n", allTasks[taskNumToUnmark].getStatusIcon(),
+                allTasks[taskNumToUnmark].getDescription());
+        drawLine(true);
+    }
+
+    private static void listItems() {
```

Comment:
The method naming `listItems` is a bit ambiguous, I believe it's better to rename it to something like `printListItems`, because we are basically printing all the items.